cli/Makefile

```diff
@@ -0,0 +1,2 @@
+test:
```

Comment:
I created a new makefile instead of using the root one because the CLI probably gonna be extracted to an individual repo later. So I want to keep it as isolated as possible.